### PR TITLE
Adding variations to store

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

- added variations to the store.

## Connected PR
https://github.com/infinum/eightshift-frontend-libs/pull/630

### CODE REVIEW

**Review by goranalkovic-infinum — DISMISSED** [review]

File: src/Helpers/StoreTrait.php [hunk 1]
```diff
@@ -162,6 +163,50 @@ static function ($item) use ($component) {
...
+	/**
+	 * Get variation details.
+	 *
+	 * @param string $variation Variation name to get.
+	 * @return array<mixed>
+	 */
+	public static function getVariation(string $variation): array
+	{
+		$variations = \array_filter(
+			self::getvariations(),
```
Comment (iobrado) [inline src/Helpers/StoreTrait.php:201]:
```suggestion
			self::getVariations(),
```

**Review by iobrado — CHANGES_REQUESTED** [review]
Noticed just one typo 🙂

**Review by iobrado — APPROVED** [review]

**Review by mbmjertan — DISMISSED** [review]

**Review by dingo-d — APPROVED** [review]

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left one comment, also check the CI failures

**Comment by iruzevic** [comment]
fixed

**Pull request merged**